### ISSUE TITLE
obs help

### ISSUE THREAD

**Issue by mikey150** [WatchBeam/ftl-studio#53]
someone help me when i click on start streaming it doesnt do anything and am using obs 0.15.4-ftl1.6

**Comment by Maximilian-Reuter** [comment]
Hello @mikey150 ;)

First of all I would suggest posting this in the appropriate repository.
Secondly, without a log file nobody is really going to be able to help you since we aren't magicians :)

Cheers Maximilian

**Comment by mikey150** [comment]
https://gist.github.com/da1afd50bf0d86b78efc220628320058

**Comment by Maximilian-Reuter** [comment]
@mikey150  See ? now we're getting somewhere ;)

The solution is easy, you have to change the sample rate to 48KHz.

**Comment by Maximilian-Reuter** [comment]
@mikey150 in case u want to also be able to stream in rtmp, have a look at my release: https://github.com/Maximilian-Reuter/obs-studio-ftl/releases/tag/0.15.4-ftl.7-complete

**Comment by mikey150** [comment]
hello i have obs with ftl and now my stream works for a bit then it goes
offline

**Comment by mikey150** [comment]
[image: Inline images 1]

**Comment by Maximilian-Reuter** [comment]
Do you have my version or the one from the beam repository ? Could you post a log?

**Comment by mikey150** [comment]
What version of beam

**Comment by sslivins** [comment]
@Maximilian-Reuter.  thanks for the great work. if you're happy with what you have I would encourage you to create a pull request into the obs-studio-ftl : ftl branch (I will be merging flow_control back into ftl this weekend) and I'll release your contribution as part of our releases going forward.

**Comment by Maximilian-Reuter** [comment]
@sslivins sure I'd love to have been able to contribute to such a great project ;)

**Comment by sslivins** [comment]
@Maximilian-Reuter glad to hear it and I'm glad you took the time to contribute.  do the pull request against the ftl branch when you're ready.  there is unlikely to be any overlap with flow_control so timing isnt critical

**Comment by Maximilian-Reuter** [comment]
@sslivins So no cherrypicking or anything required ? Just my flow_control branch into your ftl ?

**Comment by mikey150** [comment]
Someone help me plz

**Comment by Maximilian-Reuter** [comment]
@mikey150 How often do I have to tell you ? Without Logs nobody can do anything!

**Comment by sslivins** [comment]
@Maximilian-Reuter yup, that should be fine, i'll go through it but it should be fine.  it wont be the next release but should be in later this week if all goes well

**Comment by sslivins** [comment]
@mikey150 try to stream, then go to logs->upload current log and post the link here.  this isnt the right forum for streaming on beam btw, you should contact us at beam.pro/contact

**Comment by mikey150** [comment]
Well i was having trouble with obs its not beam am having trouble with
